lms/static/scripts/frontend_apps/components/JSTORPicker.js

```diff
@@ -211,6 +217,36 @@ export default function JSTORPicker({ onCancel, onSelectURL }) {
             </div>
           )}
 
+          {metadata.data && (
+            <div className="grow flex flex-col justify-end">
+              <div className="flex space-x-2 items-center px-1">
+                <div className="grow text-right">
```

Comment:
Acknowledge that flex gymnastics here more advanced than typical. Hoping that we can couple "what we've learned" as we refactor shared components and also step back and look at the content pickers as a follow-up to see if we could simplify and standardize overall.